keycloak/keycloak-admin-ui
# Realm settings(Client policies): Update client-scopes condition to match new design

## Motivation

https://issues.redhat.com/browse/CLUX-330
https://marvelapp.com/prototype/2b3ghc06/screen/81619352

## Brief Description

- Adds screen, correct fields, and functionality to client policies -> add condition -> client-scopes condition type
- Supports save and edit functionality

## Verification Steps


1. Go to Realm settings -> Client policies -> Policies subtab
2. Create a new client policy if one doesn't exist yet.
3. Click on the client policy and under Conditions, click "Add condition"
4. Select the "client-scopes" condition type.
5. Verify that the UI and functionality match the design and work as expected. 
6. Save the client-scopes condition.
7. Navigate back to the client-scope condition type and verify that updating the data works as expected.

## Checklist:


## Additional Notes

Dependent on #1532  - will rebase once merged to simplify review process.
